resources/views/character/partials/header.blade.php

```diff
@@ -38,9 +38,16 @@
             @endif
         </ul>
     </li>
-    @if ($character->raid_id || $character->class)
+    @if ($character->raid_id || $character->class || $character->is_alt)
         <li>
             <ul class="list-inline">
+                @if ($character->is_alt)
+                <li class="list-inline-item font-weight-bold">
+                    <span class="tag d-inline" style="color: orange;">
+                        Alt
+                    </span>
+                </li>
+                @endif

```

Comment:
We don't have a styleguide, but I've noticed this one twice now. Do you have any strong feelings about whether the contents of the `@if` statements are indented? It breaks the flow of the HTML for them to be indented, but it's easier to follow the logic if they are. (especially if there is a very large block of HTML inside the if statement)

My preference is to indent the HTML.

Thoughts?

Reply:
Indent is fine, I dont think it makes a huge difference either way.

Reply:
Doesn't mage a huge difference? _Are you sure?_

https://www.youtube.com/watch?v=SsoOG6ZeyUI

:P